<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetricItemsDisplayConfigForm</class>
 <widget class="QWidget" name="MetricItemsDisplayConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="devices_tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Device</string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="restore_pushButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>564</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_pushButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>564</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="deviceMetric_tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>801</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab:selected {
	background-color: rgb(114, 159, 207);
}

QTabBar::tab:!selected {
	background-color: rgb(186, 189, 182);
}</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="savina_tab">
    <attribute name="title">
     <string>Savina</string>
    </attribute>
    <widget class="QTableWidget" name="savina_tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>801</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget::indicator:checked {
	left: 100px;
}</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>新增列</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新增列</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新增列</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Metric Type</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>Type1</string>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>Test Metric</string>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>Type1</string>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>Test Metric</string>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>Type3</string>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>Test Metric</string>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="mp40_tab">
    <attribute name="title">
     <string>MP40</string>
    </attribute>
    <widget class="QTableWidget" name="mp40_tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>801</width>
       <height>321</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>新增列</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Metric Type</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>Type2</string>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>Test Metric</string>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
